install

```diff
@@ -481,14 +481,14 @@ if !mac?
   #{Tty.bold}Fedora, Red Hat, CentOS, etc.#{Tty.reset}
     sudo yum groupinstall 'Development Tools'
   See #{Tty.underline}http://linuxbrew.sh/#dependencies#{Tty.reset} for more information.
-- Add Linuxbrew to your #{Tty.underline}#{shell_profile}#{Tty.reset} by running
+- Add the brew command to your #{Tty.underline}#{shell_profile}#{Tty.reset} by running
```

Comment:
```suggestion
- Configure Brew in your #{Tty.underline}#{shell_profile}#{Tty.reset} by running
```
or `Linuxbrew` or `Homebrew`. Since this is `Linuxbrew/install`, I'm fine with `Linuxbrew.